### PR TITLE
Fix entry points

### PR DESCRIPTION
**Pull request by xylar**
Add tests for all 3 entry points

Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
You should always rerender, I think.  The build system might have changed (e.g. different keys for CI) or something completely unrelated to the recipe itself.  Pinnings might also be updated.  All of that requires rerendering.  Always better safe than sorry.  But I was pretty sure that nothing would happen since I just rerendered yesterday.

**Comment by xylar** [comment]
Also, the review request happens automatically via the bot, so no need to feel like you actually have to review these PRs.

**Comment by jhkennedy** [comment]
Looks like the fails are because of a typo: `download_nautral_earth_110m`

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,12 +11,13 @@ source:
...
 build:
-    number: 0
+    number: 1
     script: "{{ PYTHON }} -m pip install . --no-deps -vv"
     noarch: python
     entry_points:
         - mpas_analysis = mpas_analysis.__main__:main
-        - download_analysis_data = mpas_analysis.__main__:download_analysis_data
+        - download_analysis_data = mpas_analysis.download_data:download_analysis_data
+        - download_nautral_earth_110m = mpas_analysis.download_data:download_nautral_earth_110m
```
Comment (jhkennedy) [inline recipe/meta.yaml]: typo in "natural" ; x2 here

File: recipe/meta.yaml [hunk 2]
```diff
@@ -53,6 +54,9 @@ test:
     - pytest
   commands:
     - pytest --pyargs mpas_analysis
+    - mpas_analysis --help
+    - download_analysis_data --help
+    - download_nautral_earth_110m --help
```
Comment (jhkennedy) [inline recipe/meta.yaml]: typo in "natural" here too

Comment (xylar) [reply recipe/meta.yaml]: Copy-paste at its worst!  The typo is also in `setup.py` but everyone uses `conda-forge` so I don't think that's as big a deal.  I'll fix it but I won't do a new release for it...

Comment (jhkennedy) [reply recipe/meta.yaml]: Yeah, This one was hard to see -- I just read right over it multiple times.

Seems fair to wait on a bugfix release for it to actually break on someone if you expect everyone to be using conda-forge. :-D

**Pull request merged**